src/Commands/SpalpCommands.php

```diff
@@ -88,4 +88,85 @@ public function importJson($module = '') {
     ]));
   }
 
+  /**
+   * Show the difference between config on the applanding node and in JSON.
+   *
+   * @param string $module
+   *   The machine name of the module.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @throws \Drush\Exceptions\UserAbortException
+   *
+   * @command spalp:config-diff
+   * @aliases scd
+   */
+  public function configDiff($module = '') {
+    // If we don't have a module from the command, ask for one.
+    if (empty($module)) {
+      $modules = $this->spalpConfig->getAppIds();
+      $module = $this->io()->choice('Which module would you like to import configuration for?', $modules);
+    }
+
+    $json = $this->spalpCore->getConfigFromJson($module);
+    if (empty($json)) {
+      throw new \Exception(dt('@module does not provide JSON configuration.', [
+        '@module' => $module,
+      ]));
+    }
+
+    $node = $this->spalpCore->getAppNode($module);
+    if (empty($node)) {
+      throw new \Exception(dt('There is no app landing node for @module.', [
+        '@module' => $module,
+      ]));
+    }
+
+    $diff = $this->spalpCore->getAppConfigDiff($module);
+    $this->logger()->success(dt('Configuration diff between node @nid and JSON for @module: @diff', [
+      '@nid' => $node->id(),
+      '@module' => $module,
+      '@diff' => print_r($diff, 1),
+    ]));
+
+  }
+
+  /**
+   * Output the current config on the applanding node for a module.
+   *
+   * @param string $module
+   *   The machine name of the module.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   * @throws \Drush\Exceptions\UserAbortException
+   *
+   * @command spalp:config-export
+   * @aliases sce
+   */
+  public function configExport($module = '') {
+    // If we don't have a module from the command, ask for one.
+    if (empty($module)) {
```

Comment:
A lot of this is similiar/indentical to configDiff - any benefit of a helper function for this?

Reply:
yes I was wondering the same, but I can't think of an elegant way to do it